### PR TITLE
CHANGELOG: add AOPP and BitBox02 bootloader screen changes

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## [Unreleased]
+- Add support for the address ownership proof protocol (AOPP), i.e.: handle 'aopp:?...' URIs in the BitBoxApp.
```
Comment (x1ddos) [inline CHANGELOG.md:4]: Add a link to https://aopp.group?
Also, normally spelled upper case Address Ownership ...

**Review by x1ddos — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (x1ddos) [reply CHANGELOG.md:4]: also, " ... in the BitBoxApp" - this **is** the app changelog :)

**Comment by benma** [comment]
@x1ddos added more forgotten changes, ptal.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,14 @@
 # Changelog
 
 ## [Unreleased]
+- Add support for the Address Ownership Proof Protocol (AOPP), i.e.: handle 'aopp:?...' URIs. See https://aopp.group/.
 - Add a guide entry: How to import my transactions into CoinTracking?
 - Updated to Qt 5.15 from Qt 5.12 for Linux, macOS and Windows
 - Revamped account-info view to show account keypath, scriptType etc.
 - Allow disabling accounts in 'Manage accounts'.
 - Prevent screen from turning off while the app is in foreground on Android
+- Allow entering the BitBox02 startup settings in 'Manage device'
```
Comment (x1ddos) [inline CHANGELOG.md:10]: Maybe expand on what it's for? "Allow entering ... to toggle show firmware hashes at any time"

**Review by x1ddos — APPROVED** [review]
Nice. I forgot about those too.

File: CHANGELOG.md (hunk 2 above)
Comment (benma) [reply CHANGELOG.md]: done

**Pull request merged**